src/scripts/main.js

```diff
@@ -1,3 +1,201 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const tbody = document.querySelector('tbody');
+const error = document.createElement('div');
+const savingObject = {
+  state: true,
+};
+
+error.classList.add('error');
+error.classList.add('notification');
+error.setAttribute('data-qa', 'notification');
+
+const success = document.createElement('div');
+
+success.classList.add('success');
+success.classList.add('notification');
+success.innerHTML = 'You were successfully added to the table';
+success.setAttribute('data-qa', 'notification');
+
+function convertFromForm(number) {
+  return '$' + parseInt(number).toLocaleString('de-DE').split('.').join(',');
+}
+
+function converter(number) {
+  return number.slice(1).split(',').join('.');
+}
+
+body.insertAdjacentHTML('beforeend', `
+<form class="new-employee-form" action="">
+  <label  for="name">Name:
+    <input required data-qa="name" type="text">
+  </label>
+  <label for="">Position:
+    <input required data-qa="position" type="text">
+  </label>
+  <label for="">Office:
+    <select required data-qa="office" name="" id="">
+      <option selected value="value1">Tokyo</option>
+      <option value="value2">Singapore</option>
+      <option value="value3">London</option>
+      <option value="value3">New York</option>
+      <option value="value3">Edinburgh</option>
+      <option value="value3">San Francisco</option>
+    </select>
+  </label>
+  <label for="">Age:
+    <input required data-qa="age" type="number">
+  </label>
+  <label for="">Salary:
+    <input required data-qa="salary" type="number">
+  </label>
+  <button class="new-employee-form__button" type="submit">Save to table</button>
+</form>`);
+
+const button = document.querySelector('.new-employee-form__button');
+
+function tableSort(element, order) {
+  const trs = document.querySelectorAll('tbody tr');
+  const parentNode = element.parentNode;
+  const position = [...parentNode.children].indexOf(element);
+
+  let sorted = [...trs]
+    .sort((a, b) => a.children[position]
+      .innerHTML > b.children[position].innerHTML ? 1 : -1);
+
+  if (position === 4) {
+    sorted = [...trs]
+      .sort((a, b) => converter(a.children[position]
+        .innerHTML) - converter(b.children[position].innerHTML));
+  }
+
+  if (order === false) {
+    sorted = [...trs]
+      .sort((a, b) => b.children[position]
+        .innerHTML > a.children[position].innerHTML ? 1 : -1);
+
+    if (position === 4) {
+      sorted = [...trs]
+        .sort((a, b) => converter(b.children[position]
+          .innerHTML) - converter(a.children[position].innerHTML));
+    }
+  }
```

Comment:
Code dublicated. Fix it.

```suggestion
  if (position === 4) {
    sorted = [...trs]
      .sort((a, b) => converter(a.children[position]
        .innerHTML) - converter(b.children[position].innerHTML));
  }

  if (order === false) {
    sorted = [...trs]
      .sort((a, b) => b.children[position]
        .innerHTML > a.children[position].innerHTML ? 1 : -1);

    if (position === 4) {
      sorted = [...trs]
        .sort((a, b) => converter(b.children[position]
          .innerHTML) - converter(a.children[position].innerHTML));
    }
  }
```

Reply:
i dont understand, the code is not dublicated, if false AND if position === 4 i need the table to be sorted in desc order